### PR TITLE
dmesg updates

### PR DESCRIPTION
**Pull request by perfecto25**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by schneiderl** [comment]
Should that tab exist on the start of the example lines?

File: pages/linux/dmesg.md [hunk 1]
```diff
@@ -4,20 +4,32 @@
...
 
 - Show how much physical memory is available on this system:
 
-`dmesg | grep -i memory`
+    `dmesg | grep -i memory`
 
 - Show kernel messages 1 page at a time:
 
-`dmesg | less`
+    `dmesg | less`
```
Comment (sbrl) [inline pages/linux/dmesg.md]: Could you remove the spaces at the beginning of every line please?

File: pages/linux/dmesg.md [hunk 2]
```diff
@@ -4,20 +4,32 @@
...
+
+- Show kernel messages with a timestamp (available in kernels 3.5.0 and newer):
```
Comment (sbrl) [inline pages/linux/dmesg.md]: I'm unsure that we need to include `(available in kernels 3.5.0 and newer)` here. Are there any current Linux distributions that use a kernel older than this in their latest release?

Comment (perfecto25) [reply pages/linux/dmesg.md]: kernels like  2.6.32-642.4.2.el6.x86_64  dont have these options, centos 6 for example which is still fairly common

Comment (sbrl) [reply pages/linux/dmesg.md]: ....I see. Checking the Centos website reveals that there's still a year or two of official support left there, so this should be ok then :-)

It just surprised me - I haven't run into a 2.6.x kernel for _ages_! (The last one was the OVH VPS kernel - ewwwwwww........)

**Pull request merged**